man/org.freedesktop.systemd1.xml

```diff
@@ -2565,6 +2565,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       readonly u NRestarts = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly s OOMPolicy = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly a(ssx) OpenFile = [...];
```

Comment:
You should always use an unsigned 64-bit integer for bitflags, so this needs to be `a(sst)`